src/SkeletalTrapezoidation.cpp

```diff
@@ -645,7 +641,7 @@ void SkeletalTrapezoidation::collapseSmallEdges(coord_t snap_dist)
 }
 
 void SkeletalTrapezoidation::fixNodeDuplication()
-{ // fix duplicate verts
```

Comment:
This got removed instead of capitalized.

Reply:
Correct. Because the name fixNodeDuplication is quite literally already saying what is going on here ;)